src/Components/Test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TestImage from 'Images/Test.png';
+import TestSVG from 'Images/test.svg';
```

Comment:
I am highlighting the issue here because it was where I got the eslint error, but the issue is on the folder name "Images" that was not updated to be lowercase.
Please before fixing try to figure out why husky is not blocking your commit and detecting this errors.
<img width="1151" alt="Screenshot 2022-02-23 at 15 06 20" src="https://user-images.githubusercontent.com/75747772/155348123-d9b06de2-ec41-4c86-8728-49acac81ee69.png">
<img width="855" alt="Screenshot 2022-02-23 at 15 15 22" src="https://user-images.githubusercontent.com/75747772/155348541-d0ace22c-9b61-4712-981f-a4f90e7ad841.png">